src/commands/createContainerApp/TargetPortStep.ts

```diff
@@ -24,6 +24,8 @@ export class TargetPortStep extends AzureWizardPromptStep<IContainerAppContext>
         const num = Number(val);
         if (isNaN(num)) {
             return localize('enterNumber', 'Enter a valid port number')
+        } else if (!Number.isInteger(val)) {
```

Comment:
Oh, missed this:
```suggestion
        } else if (!Number.isInteger(num)) {
```

Reply:
lol sorry, I should have been paying more attention myself